lib/epics/client.rb

```diff
@@ -245,4 +245,7 @@ def setup_cipher(method, passphrase, salt)
     cipher.key = OpenSSL::PKCS5.pbkdf2_hmac_sha1(passphrase, salt, 1, cipher.key_len)
   end
 
+  def verify_ssl?
+    ENV['EPICS_VERIFY_SSL'] != 'false'
```

Comment:
scary! :) 
maybe we should log a warning in case verification is disabled? trying to prevent people from doing stupid things.